client/my-sites/scan/history/index.tsx

```diff
@@ -18,7 +18,7 @@ interface Props {
 	showNavigation?: boolean;
```

Comment:
We can remove this unused prop from the type too
```suggestion
```

Reply:
Nice catch 🙏